pkgs/development/libraries/qpdf/default.nix

```diff
@@ -47,6 +48,8 @@ stdenv.mkDerivation rec {
   doCheck = true;
 
   passthru.tests = {
+    pkg-config = testers.hasPkgConfigModules { package = finalAttrs.finalPackage; };
+    version = testers.testVersion { package = finalAttrs.finalPackage; };
```

Comment:
I'd vote for using the new `versionCheckHook` if possible.

Reply:
thanks! i hadn't come across that yet

Reply:
> thanks! i hadn't come across that yet

I'm glad you liked it! (I wrote it)